deployments/EKS/kubearmor.yaml

```diff
@@ -23,312 +25,322 @@ metadata:
   name: kubearmor
   namespace: kube-system
 spec:
-  selector:
-    kubearmor-app: kubearmor-relay
   ports:
   - port: 32767
     protocol: TCP
+    targetPort: 0
```

Comment:
Didnt understand the rationale of setting the `targetPort: 0` ... Apparently the pod should also wait on the `targetPort`. Not sure what happens if the port is set to 0.

Reply:
https://github.com/daemon1024/KubeArmor/blob/gen-deployment/deployments/objects.go#L66-L71
I didn't set the targetPort(as in the original YAML) so it takes the default value from the spec I believe.
Not sure what's the significance. Looking into it.

Reply:
If targetPort is not set, it defaults to being same as "Port", Marking it as 0 is misleading, so I will set it to 32767